waku/v2/node/wakunode2.nim

```diff
@@ -172,7 +172,8 @@ proc new*(T: type WakuNode, nodeKey: crypto.PrivateKey,
     wakuFlags = none(WakuEnrBitfield),
     nameResolver: NameResolver = nil,
     sendSignedPeerRecord = false,
-    dns4DomainName = none(string)
+    dns4DomainName = none(string),
+    discv5UdpPort = none[Port]()
```

Comment:
Nitpick/curiosity: Why are you defining these `none` values using two different notations? 

From my experience, code style consistency is an important aspect of codebase maintainability. And it is usually overlooked. If all the statements are uniform, it is easier to detect things that are not "correct" (e.g.bugs).

In this case: IMHO we should be consistent and use just one format. I like the `none(Port)` format, over the generic parameter format.

Reply:
Not at all nitpicky and in fact a very important principle. Did this to be consistent with the way the "other" port and IP address options are initialised in https://github.com/status-im/nwaku/blob/9a16883012cabc84a8f7793eed6f486fc85cbc46/waku/v2/node/wakunode2.nim#L164, but then missed the inconsistency with the parameter right above this line. I've now changed everything to use the `none(type)` format, which I also prefer ([f2fa3ef](https://github.com/status-im/nwaku/pull/996/commits/f2fa3efe9f1a1809a09235973aef1cdcf97e7707)). Thanks for pointing this out.